build.yaml

```diff
@@ -0,0 +1,22 @@
+builders:
+  resourceBuilder:
+    import: "../../../tool/builder.dart"
```

Comment:
cc @jakemac53 

This import can be `import: "tool/builder.dart"` after a recent change